### ISSUE TITLE
google_storage_bucket returns no price for EU

### ISSUE THREAD

**Issue by alikhajeh1** [infracost/infracost#451]
The following resource returns no prices, but according to [this](https://cloud.google.com/storage/pricing) the EU multi-region has prices:
```
  NAME                                                       MONTHLY QTY  UNIT            PRICE   HOURLY COST  MONTHLY COST

  google_storage_bucket.my_storage_bucket
  ├─ Storage (standard)                                                -  GiB-months      0.0000            -             -
  ├─ Object adds, bucket/object list (class A)                         -  10k operations  0.0000            -             -
  └─ Object gets, retrieve bucket/object metadata (class B)            -  10k operations  0.0000            -             -
  Total                                                                                                     -             -

  OVERALL TOTAL (USD)                                                                                  0.0000        0.0000
```

Terraform resource:
```
resource "google_storage_bucket" "test" {
  name          = "test"
  location      = "EU"
  force_destroy = false
}
```

CLI logs:
```
time="2021-02-19T08:28:44Z" level=debug msg="Falling back to default region (us-central1) for google_storage_bucket.my_storage_bucket"
time="2021-02-19T08:28:44Z" level=debug msg="Loading usage data from Infracost provider resources"
time="2021-02-19T08:28:44Z" level=info msg="starting: Calculating cost estimate"
time="2021-02-19T08:28:44Z" level=debug msg="Getting pricing details from https://pricing.api.infracost.io/graphql for google_storage_bucket.my_storage_bucket"
time="2021-02-19T08:28:45Z" level=warning msg="No products found for google_storage_bucket.my_storage_bucket Storage (standard), using 0.00"
time="2021-02-19T08:28:45Z" level=warning msg="Multiple prices found for google_storage_bucket.my_storage_bucket Object adds, bucket/object list (class A), using the first price"
time="2021-02-19T08:28:45Z" level=warning msg="Multiple prices found for google_storage_bucket.my_storage_bucket Object gets, retrieve bucket/object metadata (class B), using the first price"
```

**Comment by sinabakh** [comment]
I'm looking into it.

**Comment by alikhajeh1** [comment]
Released in v0.8, migration guide is [here](https://www.infracost.io/docs/guides/v0.8_migration).
